kubernetes/charts/weblogic-operator/templates/_operator-clusterrole-general.tpl

```diff
@@ -28,7 +28,7 @@ rules:
   verbs: ["get", "list", "watch", "create", "update", "patch"]
 {{- end }}
 - apiGroups: ["weblogic.oracle"]
-  resources: ["domains", "domains/status"]
+  resources: ["clusters", "domains", "domains/status"]
```

Comment:
You will need "clusters/status" here as well. This will be required once you have the operator updating status for the Cluster resources.

Reply:
right, I can add it but I was waiting till I actually started work on the cluster status.